DQM/GEM/interface/GEMDQMBase.h

```diff
@@ -237,13 +299,16 @@ class GEMDQMBase : public DQMEDAnalyzer {
 
     dqm::impl::MonitorElement *FindHist(K key) {
       if (mapHist.find(key) == mapHist.end()) {
-        std::cout << "" << std::endl;  // FIXME: It's about sending a message
+        std::cout << "WARNING: Cannot find the histogram corresponing to the given key"
```

Comment:
I am sorry but couts are discouraged, please use edm::Log or similar instead

Reply:
Oops, okay. I'll fix it.